atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1317,7 +1317,9 @@ void WebContents::MediaStartedPlaying(const MediaPlayerInfo& media_info,
 }
 
 void WebContents::MediaStoppedPlaying(const MediaPlayerInfo& media_info,
-                                      const MediaPlayerId& id) {
+                                      const MediaPlayerId& id,
+                                      WebContentsObserver::MediaStoppedReason
+                                      reason) {
   Emit("media-paused");
```

Comment:
we should pass the reason in the emit - cc @darkdh

Reply:
`kReachedEndOfStream` is helpful, I will do this in follow up https://github.com/brave/muon/issues/446